cmd/options.go

```diff
@@ -61,6 +60,7 @@ func optionFlagSet() *pflag.FlagSet {
 	flags.Bool("insecure-skip-tls-verify", false, "skip verification of TLS certificates")
 	flags.Bool("no-connection-reuse", false, "disable keep-alive connections")
 	flags.Bool("no-vu-connection-reuse", false, "don't reuse connections between iterations")
+	flags.Duration("min-iteration-duration", 0, "minimum amount of time a single iteration should take")
```

Comment:
I don't think it becomes apparent that if it's less k6 will wait for the difference maybe "minimum amount of time k6 will take executing a single iteration" or more verbosely "minimum amount of time a single iteration will take. k6 will wait for the difference if shorter"